examples/MagicWeather/README.md

```diff
@@ -18,7 +18,7 @@ See minimum platform version requirements for RevenueCat's *Purchases* SDK [here
 
 | Feature                          | Sample Project Location                   |
 | -------------------------------- | ----------------------------------------- |
-| 🕹 Configuring the *Purchases* SDK  | [MainActivity.kt](app/src/main/java/com/revenuecat/sample/MainActivity.kt) |
+| 🕹 Configuring the *Purchases* SDK  | [MainActivity.kt](app/src/main/java/com/revenuecat/sample/MainApplication.kt) |
```

Comment:
good catch! do you mind updating the link text as well? :)

Reply:
Done!